### ISSUE TITLE
Montserrat "regular" font style too strong

### ISSUE THREAD

**Issue by cleverworx** [majodev/google-webfonts-helper#75]
The "regular" font style is way too strong, looks a lot closer to the 600 style.

**Comment by sntran** [comment]
Same with IBM Flex Sans.

On the helper site:

![image](https://user-images.githubusercontent.com/973206/55247962-da7e9e00-5216-11e9-8704-d5b4d013b1e1.png)

Versus on Google:

![image](https://user-images.githubusercontent.com/973206/55247978-e66a6000-5216-11e9-961b-4fe7a02ccfa9.png)

**Comment by JordyValentine** [comment]
@cleverworx I agree the font is a bit heavier than it should be, but are you sure it's not an issue with the font itself?
Monsterrat has had some issue with font weight in the past, they've changed the font a couple of times over the last few years. It seem's sometime ago the Monsterrat font had it's spacing/weight etc changed, making the default 400 closer to 500. You can see a comparison in this issue: [https://github.com/JulietaUla/Montserrat/issues/60#issuecomment-342628595]

Have you tried playing around with the font weight or even an older version of Monsterrat?
